jobs/build-arch.Jenkinsfile

```diff
@@ -306,6 +306,14 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
             pipeutils.build_artifacts(pipecfg, params.STREAM, basearch)
         }
 
+        // secex specific tests
+        if (shwrapCapture("cosa meta --get-value images.qemu-secex") != "None") {
+            stage("Kola:TestSecex") {
```

Comment:
Minor: maybe just `Kola:Secex`? "Test" is implied by kola. "Kola:TestISO" below is a reference to the command name.